packages/driver/src/cypress/cy.ts

```diff
@@ -244,13 +246,14 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
     this.reset = this.reset.bind(this)
     this.addCommandSync = this.addCommandSync.bind(this)
     this.addCommand = this.addCommand.bind(this)
+    this.addQuery = this.addQuery.bind(this)
     this.now = this.now.bind(this)
-    this.replayCommandsFrom = this.replayCommandsFrom.bind(this)
     this.onBeforeAppWindowLoad = this.onBeforeAppWindowLoad.bind(this)
     this.onUncaughtException = this.onUncaughtException.bind(this)
     this.setRunnable = this.setRunnable.bind(this)
     this.cleanup = this.cleanup.bind(this)
     this.setSubjectForChainer = this.setSubjectForChainer.bind(this)
+    this.currentSubject = this.currentSubject.bind(this)
```

Comment:
Do we need to add this to the global object?  If so, we should double check this is getting set to null on a refresh / when cypress is town down when navigating between specs in open mode so the browser will gc.

Reply:
I added them here to follow the existing pattern where all methods on `cy` are bound to `this`.

I don't think it makes any sense to add tests here - binding a method to ensure `this` is always consistent isn't something that requires cleaning up between tests.